0.1.1 (#44)
MetaMask/action-create-release-pr: CHANGELOG.md

## [Unreleased]

> ## [0.1.1]
> ### Fixed
> - This action ([#42](https://github.com/MetaMask/action-create-release-pr/pull/42))
>   - Due to a poorly formatted Bash invocation in `action.yml`, the Action was completely broken.

Reviewer: ```suggestion
## [0.1.1]
### Fixed
- This action ([#42](https://github.com/MetaMask/action-create-release-pr/pull/42))
  - Due to a poorly formatted Bash invocation in `action.yml`, the Action was completely broken.

```

## [Unreleased]

## [0.1.1]
### Fixed
- This action ([#42](https://github.com/MetaMask/action-create-release-pr/pull/42))

>   - Due to a poorly formatted Bash invocation in `action.yml`, the Action was completely broken.

Reviewer: ```suggestion
  - Due to a poorly formatted Bash invocation in `action.yml`, this Action was completely broken.
```

## [Unreleased]

## [0.1.1]

> ### Uncategorized
> - Use local version of Action for create-release-pr workflow ([#43](https://github.com/MetaMask/action-create-release-pr/pull/43))
> - Fix multi-line bash invocation in action.yml ([#42](https://github.com/MetaMask/action-create-release-pr/pull/42))
> - Update local workflow MetaMask Action versions ([#41](https://github.com/MetaMask/action-create-release-pr/pull/41))
> 

Reviewer: ```suggestion
### Fixed
- This action ([#42](https://github.com/MetaMask/action-create-release-pr/pull/42))
  - Due to a poorly formatted Bash invocation in `action.yml`, the Action was completely broken.
```
